taxonomies/food/categories.txt

```diff
@@ -1141,6 +1164,7 @@ hr:Pića i pripravci za piće
 it:Bevande e preparati per bevande
 nl:Dranken en drankbereidingen
 description:en:This mixes ready to drink beverages and not-ready to drink beverages
+opposite:en: en:meals
```

Comment:
The opposite of "Beverages" would be "Food" I think, not "meals". Of course that would mean putting most parent categories under "Food"